includes/TCPListener.hpp

```diff
@@ -21,17 +21,26 @@ class TCPListener
 {
     int _epfd;
     std::vector<std::pair<int, const char*> > _fd_port_list;
+    std::vector<TCPStream>  _connected_client;
```

Comment:
A manager class could be more convenient.

If you had to send the whole connected clients list, including their methods, you would need to send the whole TCPListener. Whereas a manager allows you to use this list regardless from the server it belongs to.